civihr_employee_portal/views/includes/civihr_employee_portal_handler_aggregated_personal_address.inc

```diff
@@ -84,4 +85,62 @@ class civihr_employee_portal_handler_aggregated_personal_address extends views_h
       filter_xss(ArrayHelper::value('country_id.name', $address))
     );
   }
+
+  /**
+   * Returns the HTML for the address as it will appear in the view.
+   *
+   * @param array $address
+   *   The address data
+   * @return string
+   *   The formatted address
+   */
+  private function formatAddressWithLabels($address) {
+    $format_street_address = '
+    <div class="views-field views-field-street-address">
+      <span class="views-label views-label-street-address">Home Address</span>
+      <span class="field-content">
+        %s
+        %s
+        %s
+      </span>
+    </div>
+    ';
+
+    $format_postcode = '
+    <div class="views-field views-field-postal-code">
+      <span class="views-label views-label-postal-code">Postcode</span>
+      <span class="field-content">%s</span>
+    </div>
+    ';
+
+    $format_country = '
+    <div class="views-field views-field-country">
+      <span class="views-label views-label-country">Country</span>
+      <span class="field-content">%s</span>
+    </div>
+    ';
+
+    $street_address = filter_xss(ArrayHelper::value('street_address', $address));
```

Comment:
Wouldn't it be easier to call `filter_xss` just once on the final string?

Reply:
@mickadoo no because it will remove also other HTML structure or properties.

Reply:
@mickadoo You were right :+1:  thanks